routes/forums.js

```diff
@@ -144,12 +144,29 @@ router.get('/:forum_id/', (req, res) => {
 
 // Update specific fields of forum
 router.put('/:forum_id', (req, res) => {
-
+    // TODO Confirm user, confirm changes
+    let forumId = req.params.forum_id;
+    let title = req.body.title;
+    let content = req.body.content;
+    let labels = req.body.labels;
+    forumsData.updateForum(forumId, title, content, labels)
+        .then((forumData) => {
+            return res.json({});
+        }).catch((err) => {
+            return res.status(500).json({ error: err });
+        });
 });
 
 // Delete a forum
 router.delete('/:forum_id', (req, res) => {
-
+    // TODO Confirm User
+    let forumId = req.params.forum_id;
+    forumsData.deleteForum(forumId)
+        .then((forumData) => {
+            return res.json({'redirect': '/forums'});
```

Comment:
Maybe redirect is '/' (the landing page?)

Reply:
Done.